### PR TITLE
Set TX_TOKEN in github workflow and assume it's set in pull_translations

### PR DESCRIPTION
**Pull request by suricactus**
There is one last use of `CI_SECURE_ENV_VARS`, but I don't know whether these secrets are present in GitHub:
 $encrypted_c158cd588669_key -iv $encrypted_c158cd588669_iv 

They are used in this script:
https://github.com/opengisch/QField/blob/a7e4e95cb4e652763a4baddd7119b5400984e877/scripts/ci/setup_signing_key.sh#L4-L6

Which is called from here:
https://github.com/opengisch/QField/blob/a7e4e95cb4e652763a4baddd7119b5400984e877/.github/workflows/continuous_integration.yml#L21

@m-kuhn help?

Fixes https://github.com/opengisch/QField/issues/1381

### CODE REVIEW

**Comment by m-kuhn** [comment]
This can be removed (also the encrypted files themselves, they were travis specific)

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-31276d241a16d4a7f137fc9628b35ea02f020a4c-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-31276d241a16d4a7f137fc9628b35ea02f020a4c-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-31276d241a16d4a7f137fc9628b35ea02f020a4c-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-31276d241a16d4a7f137fc9628b35ea02f020a4c-x86.apk)

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-d2650e2e854c2f7a47b359a48db1aa1bd7b442b2-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-d2650e2e854c2f7a47b359a48db1aa1bd7b442b2-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-d2650e2e854c2f7a47b359a48db1aa1bd7b442b2-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-d2650e2e854c2f7a47b359a48db1aa1bd7b442b2-x86.apk)

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-5334cbd8539e41b4a177a64a47cbc41dc59a754a-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-5334cbd8539e41b4a177a64a47cbc41dc59a754a-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-5334cbd8539e41b4a177a64a47cbc41dc59a754a-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-5334cbd8539e41b4a177a64a47cbc41dc59a754a-x86.apk)

**Comment by suricactus** [comment]
@m-kuhn this should do the trick for green master

**Comment by m-kuhn** [comment]
```
  ./scripts/ci/pull_translations.sh: line 8: tx: command not found
```

https://github.com/opengisch/QField/runs/1359974031?check_suite_focus=true#step:5:105

We'll need a `pip install transifex-client` somewhere

**Review by m-kuhn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-9c91900cdb3b94e5c110a42a73cb3ccb996ec43d-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-9c91900cdb3b94e5c110a42a73cb3ccb996ec43d-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-9c91900cdb3b94e5c110a42a73cb3ccb996ec43d-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-9c91900cdb3b94e5c110a42a73cb3ccb996ec43d-x86.apk)

File: scripts/ci/pull_translations.sh [hunk 1]
```diff
@@ -4,6 +4,9 @@ set -e
 
 echo "::group::tx-pull"
 
+echo "$TX_TOKEN" | md5sum
+echo "" | md5sum
```
Comment (m-kuhn) [inline scripts/ci/pull_translations.sh:8]: Nice idea

**Comment by m-kuhn** [comment]
For reference https://github.com/opengisch/QField/runs/1362836479#step:4:23

**Comment by m-kuhn** [comment]
Another idea https://github.com/citra-emu/citra/blob/2c0bd0f2a120144d0b7845f4e8d56fb243ba36ef/.travis/transifex/docker.sh#L3-L9

File: scripts/ci/pull_translations.sh [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+echo "${BASH_SOURCE[0]}"
+echo "$DIR"
+source ${DIR}/../version_number.sh
+pwd
+
+tx pull --all --force
```
Comment (m-kuhn) [inline scripts/ci/pull_translations.sh:16]:
```suggestion
tx --version
tx pull --all --force
```

File: scripts/ci/pull_translations.sh [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
+echo "" | md5sum
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+echo "${BASH_SOURCE[0]}"
+echo "$DIR"
+source ${DIR}/../version_number.sh
+pwd
+
+tx --version
+tx pull --all --force
```
Comment (m-kuhn) [inline scripts/ci/pull_translations.sh:17]:
```suggestion
TX_TOKEN=123 tx pull --all --force
```

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-cd250672698b2c2a6502c0f1dd31035f70fed311-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-cd250672698b2c2a6502c0f1dd31035f70fed311-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-cd250672698b2c2a6502c0f1dd31035f70fed311-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-cd250672698b2c2a6502c0f1dd31035f70fed311-x86.apk)

**Comment by qfield-fairy** [comment]
Ta-daaa, freshly created APKs are available:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-a5d67c7e530cf901d70d21f961b01b5896bf5bf4-arm64_v8a.apk)

Other architectures:

  - [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-a5d67c7e530cf901d70d21f961b01b5896bf5bf4-armv7.apk)
  - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-a5d67c7e530cf901d70d21f961b01b5896bf5bf4-x86_64.apk)
  - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1388-a5d67c7e530cf901d70d21f961b01b5896bf5bf4-x86.apk)

**Pull request merged**